src/npe2/types.py

```diff
@@ -60,7 +60,7 @@ def __array__(self) -> "np.ndarray":
 WidgetCreator = Callable[..., Widget]
 
 # ReaderContribution.command must point to a ReaderGetter
-ReaderFunction = Callable[[PathOrPaths], List[LayerData]]
+ReaderFunction = Callable[[PathOrPaths], List["LayerData"]]
```

Comment:
I don't think we want to change the type aliases here.

Reply:
https://github.com/napari/npe2/pull/340/commits/c7236b24597b9be0b7c4758a2acf5f5dff797dca